test/runtests.jl

```diff
@@ -11,11 +11,23 @@ CUDA.allowscalar(false)
     # https://github.com/JuliaReinforcementLearning/ReinforcementLearning.jl/issues/551
     @testset "1D with 0d data" begin
         b = CircularArrayBuffer{Int}(3)
-        push!(b, zeros(Int, ()))
+        append!(b, zeros(Int, ()))  # !!! not push!
```

Comment:
In https://github.com/JuliaReinforcementLearning/ReinforcementLearning.jl/issues/551 we should also use `append!` instead of `push!`